src/main/scala/gen/gmos/GmosLongslitMath.scala

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2019 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package basic.gen.gmos
+
+import basic.syntax.all._
+import gem.enum.GmosDetector.HAMAMATSU
+import gem.enum._
+import gem.math.{Angle, Wavelength}
+
+/**
+ * GMOS Longslit calculations for use in sequence generation.
+ */
+private[gmos] trait GmosLongslitMath {
```

Comment:
I separated this out because the same operations will apply equally well to GMOS South Longslit.